client/src/utils/indexs.js

```diff
@@ -1,4 +1,8 @@
 import { Button } from "@mui/material";
+import { useTranslation } from 'react-i18next';
+
+// import i18n (needs to be bundled ;)) 
```

Comment:
This comment is unclear, if something is required, it should be done in this PR. Can you explain more the purpose of this ?

Reply:
It's copypasta from the guide: https://react.i18next.com. I'll remove it.